src/openloco/map/tile.cpp

```diff
@@ -189,4 +196,56 @@ namespace openloco::map
 
         return coordinate2D;
     }
+
+    xy32 mapWindowPosToLocation(map_pos pos)
```

Comment:
Move these two functions to the map window they are exclusively used by the map window and no window else. We don't want to accidentally use these functions.